srcpkgs/nvidia390/template

```diff
@@ -0,0 +1,293 @@
+
+	vinstall libnvidia-ptxjitcompiler.so.${version} 755 usr/lib
+	ln -sf libnvidia-ptxjitcompiler.so.${version} ${DESTDIR}/usr/lib/libnvidia-ptxjitcompiler.so.1
+	ln -sf libnvidia-ptxjitcompiler.so.1 ${DESTDIR}/usr/lib/libnvidia-ptxjitcompiler.so
+
+	# nvidia-xconfig
+	vbin nvidia-xconfig
+	gzip -d nvidia-xconfig.1.gz
+	vman nvidia-xconfig.1
+
+	# nvidia-settings
+	vbin nvidia-settings
+	gzip -d nvidia-settings.1.gz
+	vman nvidia-settings.1
+	vinstall nvidia-settings.desktop 644 usr/share/applications
+	vinstall nvidia-settings.png 644 usr/share/pixmaps
+	sed -e 's:__UTILS_PATH__:/usr/bin:' \
+		-e 's:__PIXMAP_PATH__:/usr/share/pixmaps:' \
+		-i ${DESTDIR}/usr/share/applications/nvidia-settings.desktop
+
+	vinstall libnvidia-gtk3.so.${version} 755 usr/lib
+	vinstall libnvidia-gtk2.so.${version} 755 usr/lib
+
+	# application profiles (needed by nvidia-settings)
+	vinstall nvidia-application-profiles-${version}-key-documentation 644 usr/share/nvidia
+	vinstall nvidia-application-profiles-${version}-rc 644 usr/share/nvidia
+
+	# nvidia-bug-report
+	vbin nvidia-bug-report.sh nvidia-bug-report
+	vbin nvidia-debugdump
+
+	# nvidia-smi
+	vbin nvidia-smi
+	gzip -d nvidia-smi.1.gz
+	vman nvidia-smi.1
+
+	# nvidia-modprobe
+	vinstall nvidia-modprobe 4755 usr/bin
+	gzip -d nvidia-modprobe.1.gz
+	vman nvidia-modprobe.1
+
+	# License and documentation
+	vlicense LICENSE
+	vdoc README.txt README
+	vdoc NVIDIA_Changelog
+
+	# opencl pkg
+	vinstall nvidia.icd 644 etc/OpenCL/vendors
+
+	vinstall libnvidia-compiler.so.${version} 755 usr/lib
+
+	vinstall libnvidia-opencl.so.${version} 755 usr/lib
+	ln -sf libnvidia-opencl.so.${version} \
+		${DESTDIR}/usr/lib/libnvidia-opencl.so
+	ln -sf libnvidia-opencl.so.${version} \
+		${DESTDIR}/usr/lib/libnvidia-opencl.so.1
+
+	# vulkan icd
+	vinstall nvidia_icd.json 644 usr/share/vulkan/icd.d
+
+	# dkms pkg
+	vmkdir usr/src/nvidia-${version}
+	vcopy "kernel/*" usr/src/nvidia-${version}
+	vcopy ${FILESDIR}/dkms.conf usr/src/nvidia-${version}
+	sed -e "s/__PKGVER/${version}/g" \
+		-e "s/__MAKEJOBS/-j$(nproc)/g" \
+		-i ${DESTDIR}/usr/src/nvidia-${version}/dkms.conf
+	if [ "$XBPS_TARGET_MACHINE" = "i686" ]; then
+		# nvidia no longer builds "uvm" submodule for 32-bit
+		# Remove 2 lines for "nvidia-uvm" and repair consecutive enumeration following.
+		sed -e '/nvidia-uvm/,+1d' \
+ 			-e 's/\[2\]/[1]/g' \
+ 			-e 's/\[3\]/[2]/g' \
+ 			-i ${DESTDIR}/usr/src/nvidia-${version}/dkms.conf
+	fi
+
+	# Blacklist nouveau
+	vmkdir usr/lib/modprobe.d
+	echo "blacklist nouveau" > ${DESTDIR}/usr/lib/modprobe.d/nvidia.conf
+	chmod 644 ${DESTDIR}/usr/lib/modprobe.d/nvidia.conf
+
+	# Omit drm dracut module too
+	vmkdir usr/lib/dracut/dracut.conf.d
+	echo "omit_dracutmodules+=\" drm \"" > ${DESTDIR}/usr/lib/dracut/dracut.conf.d/99-nvidia.conf
+
+	vmkdir /usr/share/X11/xorg.conf.d/
+	vinstall nvidia-drm-outputclass.conf 644 /usr/share/X11/xorg.conf.d 30-nvidia-drm-outputclass.conf
+}
+
+nvidia-gtklibs_package() {
```

Comment:
This should match the source package name, `nvidia390-gtklibs`, same for the other sub-package functions.